joomla/joomla-cms
# Use API for inline script in module edit page (admin)

#### Consistently use the API

Don't echo `<script>` in the document
#### Summary of Changes

Using addScriptDeclaration() instead of echoing the script inline in the middle of the document
#### Testing Instructions

Go to admin and try to edit a module. Verify that the ordering is still functional

![screen shot 2016-07-05 at 20 57 25](https://cloud.githubusercontent.com/assets/3889375/16594467/309e5298-42f3-11e6-9beb-4057d3e3b1f3.png)
